### PR TITLE
added cookie-page

### PR DESCRIPTION
**Pull request by ph5500**

### CODE REVIEW

**Comment by samajammin** [comment]
@ph5500 FYI the build is failing:
https://app.netlify.com/sites/ethereumfoundation/deploys/5ed98f6ca05d70000750072f

File: gatsby-node.js [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+  `)
+
+  // Handle errors
+  if (result.errors) {
+    reporter.panicOnBuild(`Error while running GraphQL query.`)
+    return
+  }
+
+  result.data.allMarkdownRemark.edges.forEach(({ node }) => {
+    console.log(node)
```
Comment (samajammin) [inline gatsby-node.js:43]: ✂️

File: src/templates/static.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+import SEO from "../components/seo"
```
Comment (samajammin) [inline src/templates/static.js:4]: Wrong case in this component reference. Should be:
`import SEO from "../components/SEO"`

File: src/components/Footer.js [hunk 3]
```diff
@@ -181,22 +176,35 @@ const Footer = ({ isOpen, toggleFooter }) => {
...
               </FooterDivCanary>
               <FooterDivLinks>
-                <ImageAndTextLink
+                <OutboundLink
                   target="_blank"
                   rel="noopener noreferrer"
                   href="https://devcon.org"
                 >
-                  <img src={devconLogo} alt="Devcon Logo" />
                   Devcon.org
```
Comment (samajammin) [inline src/components/Footer.js:184]: This should just be `Devcon`, not `Devcon.org`.

File: gatsby-node.js (hunk 1 above)
Comment (ph5500) [reply gatsby-node.js:43]: fixed

File: src/templates/static.js (hunk 2 above)
Comment (ph5500) [reply src/templates/static.js:4]: fixed

File: src/components/Footer.js (hunk 3 above)
Comment (ph5500) [reply src/components/Footer.js:184]: fixed

**Pull request merged**